obelisk/cpp/lib1/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+message(STATUS "Building lib1")
+
+add_library(lib1
+        place_holder.cpp
+        include/place_holder.h)
```

Comment:
I know you said you didn't have auto-formatting ready, but can you at least manually fix these improper tabs (or at least format it with cmake and recommit)?